src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -438,24 +446,28 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
   }
 
   const handleReady = e => {
-    const paymentMethods = e.availablePaymentMethods
+    try {
+      const paymentMethods = e.availablePaymentMethods
 
-    if (paymentMethods) {
-      setHasMultiplePaymentOptions(
-        Object.values(paymentMethods).filter(Boolean).length > 1,
-      )
+      if (paymentMethods) {
+        setHasMultiplePaymentOptions(
+          Object.values(paymentMethods).filter(Boolean).length > 1,
+        )
 
-      setVisible(true)
+        setVisible(true)
```

Comment:
I'm not totally clear what these two different variables do (why are we setting one thing only if the payment methods are non-empty but another is true not matter what), but since this isn't a change in behavior I trust it is ok.